forge/test/benchmark/scripts/run_benchmark_wh.sh

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+
+# ------------------------------------------------------------------------------------------------------------------------------------------------------------ #
+# Models we run on Wormhole B0
+# ------------------------------------------------------------------------------------------------------------------------------------------------------------ #
+
+
+# MNIST Linear
+python forge/test/benchmark/benchmark.py -m mnist_linear -bs 1 -o forge-benchmark-e2e-mnist.json
```

Comment:
What is the purpose of this file? Local lab runs? 

As you already have test run defined here `.github/workflows/build-and-test.yml`, is it better to encapsulate this somehow? E.g. when you update how benchmarking model is run, you update single spot, instead of multiple :))

Reply:
The idea is to be global, but I didn't figure out way if something hangs, so for now is only local. Also, it is nice to have for manual runs.

Reply:
Also, these tests are not the same one is benchmark and the second one is pytest parameterized. In this case the first one is calling the other one, but that don't need to be the case for every model we added.